src/Poppy.js

```diff
@@ -33,8 +33,18 @@ export class Poppy {
       }
     };
     this.element = null;
+    this.init();
   }
+
   show() {
+    if (this.state.position === "topLeft" || this.state.position === "topRight") {
+      this.element.style.transform = 'translateY(10%)';
+    } else if (this.state.position === "bottomLeft" || this.state.position === "bottomRight") {
+      container.style.transform = 'translateY(0)';
```

Comment:
Transform should be applied to `this.element` instead of container

Reply:
Fixed!